src/Notepads/Resource/TransparentTextBoxStyle.xaml

```diff
@@ -6,12 +6,12 @@
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Light">
             <SolidColorBrush x:Key="CustomTextBoxBackground"
-                            Color="#E0E0E0"
+                            Color="#C0C0C0"
```

Comment:
Let's not change this for now. The color difference between mouse over and selected is visually interactive. Also the grey color does not look good in Light theme.

Reply:
I wanted to make the options button background same as the textbox background. The only way I could achieve it was by having same color for normal, mouse over and selected case. I have reverted the changes but I couldn't find a better way to sync the colors. Do you know any way it can be done??